data_validation/query_builder/random_row_builder.py

```diff
@@ -16,19 +16,29 @@
 import logging
 from typing import List
 from io import StringIO
+import sqlalchemy as sa
 import ibis
 import ibis.expr.operations as ops
 import ibis.expr.types as tz
 import ibis.expr.rules as rlz
 import ibis.backends.base_sqlalchemy.compiler as sql_compiler
+import ibis.backends.base_sqlalchemy.alchemy as sqla
 import ibis.backends.pandas.execution.util as pandas_util
+
 from ibis_bigquery import BigQueryClient
 from ibis.backends.impala.client import ImpalaClient
 from ibis.backends.pandas.client import PandasClient
 from ibis.expr.signature import Argument as Arg
 from data_validation import clients
 from data_validation.query_builder.query_builder import QueryBuilder
 
+# If you have an cx_Oracle driver installed
+try:
+    from third_party.ibis.ibis_oracle.client import OracleClient
```

Comment:
I would suggest using `clients.OracleClient` here even though you are using an import as an object in a less than natural way.

Might as well only have this whacky logic contained to one spot

Reply:
ditto for teradata